hooks/bugsnag/bugsnag.go

```diff
@@ -0,0 +1,53 @@
+package logrus_bugsnag
```

Comment:
I'm not so sure. It works in the `airbrake` hook because `tobi/airbrake-go` only has three fields to set and the hook can own that configuration, but `bugsnag/bugsnag-go` supports a lot more configuration that the calling code should own. That means the caller already has to import a package named `bugsnag`. If we make this one `bugsnag` too, they'll have to choose one to rename.